deploy-edpm.yml

```diff
@@ -30,7 +30,7 @@
 
 - name: Import run test playbook
   ansible.builtin.import_playbook: ci_framework/playbooks/08-run-tests.yml
-  when: cifmw_run_tests | default('true') | bool
+  when: cifmw_run_tests | default('false') | bool
```

Comment:
Now you need to enable it on edpm job, right? You are removing from both edpm and baremetal job.
Is edpm job passing with tempest?

Reply:
@viroel @pablintino thank you for the reviews. The job was running on edpm and end to end job and skipped on bm job. Since edpm job is running on another projects also. tempest fails there it will block the other project's pr. In order to avoid we are making this change. Once it is stabilized. We will enable it everywhere.

Reply:
ok, I didn't get that when reading PR message at the first time.